pyscript.core/src/hooks.js

```diff
@@ -45,6 +45,18 @@ export const createFunction = (self, name) => {
 const SetFunction = typedSet({ typeof: "function" });
 const SetString = typedSet({ typeof: "string" });
 
+const inputFailure = `
+    import builtins as _builtins
+    def _input(prompt=""):
+        raise Exception("\\n           ".join([
+            "input() doesn't work when PyScript runs in the main thread.",
+            "Consider using the worker attribute: https://docs.pyscript.net/2023.11.1/user-guide/workers/"
+        ]))
+
+    _builtins.input = _input
+    del _builtins
```

Comment:
why do you do `import builtins as _builtins` instead of just `import builtins`?

Reply:
as I delete that, it's a valid point indeed ... needless